jupyterhub/values.yaml

```
@@ -18,10 +18,12 @@ hub:
   consecutiveFailureLimit: 5
   activeServerLimit:
   deploymentStrategy:
-    # sqlite-pvc backed hub requires Recreate strategy to work
+    ## type: Recreate
+    ## - a sqlite-pvc backed hubs require this as that storage can only be bound
```

```suggestion
    ## - sqlite-pvc backed hubs require this as that storage can only be bound
```
otherwise we start the sentence in the singular and then switch to plural